pegasus/sites.v3/code.org/public/educate/csc/index.haml

```diff
@@ -50,74 +50,46 @@ theme: responsive_full_width
         %span •
         =hoc_s(:teacher_label_csc)
 
-%section#csc-modules
+%section.bg-neutral-light.centered
   .wrapper
-    %h2=hoc_s(:heading_find_a_module)
+    %h2=hoc_s(:csc_explore_curriculum)
+    %p=hoc_s(:csc_explore_desc)
+    %hr
+    %h3=hoc_s(:csc_find_the_right_fit)
+    .button-wrapper.centered
+      %a.link-button{href: "#csc-week-long-modules"}
+        =hoc_s(:csc_week_long_modules)
+      %a.link-button{href: "#csc-hour-long-lessons"}
+        =hoc_s(:csc_hour_long_lessons)
+
+%section#csc-week-long-modules
+  .wrapper
+    %h2=hoc_s(:csc_week_long_modules)
     %p=hoc_s(:csc_page_find_a_module_desc)
-    .action-block__wrapper.action-block__wrapper--three-col
+    = view :carousel_csc_week_long
+
```

Comment:
Let's add a `= view :section_divider_line` here between these two sections so it breaks them up a bit.

Reply:
<img width="780" alt="Screenshot 2024-02-16 at 9 17 45 AM" src="https://github.com/code-dot-org/code-dot-org/assets/37230822/b6d19059-1948-4520-9db3-fa619da1d2f9">

Reply:
Done! Good call:)